Calculator Layout iOS13/Controller/ViewController.swift

```diff
@@ -57,6 +57,9 @@ class ViewController: UIViewController {
 				displayLabel?.text = (displayLabel?.text)! + numValue
 			}
 		}
+		if displayLabel?.text == "." {
+			displayLabel?.text = "0."
```

Comment:
❓
Are there any occasions in which we'd be looking to change `.1` to `0.1`? If so, we'd want to go with a more generalizable approach

Reply:
Any text beginning with . should become 0.x. Any text ending with .0 should drop the .0.

Reply:
Confirming: we still do want the behavior that these lines currently provide, correct?